src/@next/components/organisms/CheckoutShipping/types.ts

```diff
@@ -0,0 +1,30 @@
+import { IFormError } from "@types";
+
+export interface IShippingMethodPrice {
+  /**
+   * Currency code.
+   */
+  currency: string;
+  /**
+   * Amount of money.
+   */
+  amount: number;
+}
```

Comment:
I'm not sure about it, because if we want to rely on SDK in a separate package in the future, it might hide from us those interfaces generated by Apollo in that package.